src/Illuminate/Routing/Router.php

```diff
@@ -613,7 +613,7 @@ protected function mergeGroupAttributesIntoRoute($route)
      * @param  array  $action
      * @return bool
      */
-    protected function actionReferencesController($action)
+    public function actionReferencesController($action)
```

Comment:
I think you mean overwritten as `protected` by a child class, so yeah changing this to public will cause a fatal in case anyone is overwriting the Router. Alright, thanks for the notes 👍